lib/efi.py

```diff
@@ -122,7 +122,7 @@ def get_secure_boot_guid(variable: str) -> GUID:
         'KEK': global_variable_guid,
         'db': image_security_database_guid,
         'dbx': image_security_database_guid,
-    }.get(variable)
+    }[variable]
 
 
 def cert_to_efi_sig_list(cert):
```

Comment:
shouldn't we also annotate the return value here? As it is called by `certs_to_sig_db` that has a return value I expected to also see a return value here.

Reply:
Ok in fact I just understood that you fix existing one. Forget my remark :)